tests/drone/if-diff-has-unit-tests-only.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+CHANGED_UNIT_TESTS_ONLY=True
+DIFFS=$(git diff origin/master --name-only)
+for DIFF in ${DIFFS}
+do
+	if ! echo "${DIFF}" | grep 'packages/.*/tests/.*'
+	then
+		CHANGED_UNIT_TESTS_ONLY=False
+		break
+	fi
+done
+
+if echo $CHANGED_UNIT_TESTS_ONLY | grep True
```

Comment:
```suggestion
if [ "${CHANGED_UNIT_TESTS_ONLY}" == "True" ]
```
IMO you do not need to grep for "True"